.github/workflows/programequity_slack.yml

```
@@ -0,0 +1,24 @@
+on:
+  issues:
+    types: [opened, edited]
+  issue_comment:
+    types: [created, edited]
+  pull_request:
+    types: [opened, edited, review_requested]
+  pull_request_review:
+    types: [submitted]
+  pull_request_review_comment:
+    types: [created, edited]
+
+jobs:
+  mention-to-slack:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run
+        uses: abeyuya/actions-mention-to-slack@v2
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          slack-webhook-url: ${{ secrets.SLACK_WEBHOOK_URL }}
+          icon-url: https://img.icons8.com/color/256/000000/github-2.png
+          bot-name: 'Send Mention from abeyuya/actions-mention-to-slack'
```

Let's aim for less advertising and more practicality with the bot name. 😆 

This also added what type of event the mention occurred in:

```suggestion
          bot-name: 'Mentioned on GitHub: ${{ github.event_name }}'
```